projects/packages/videopress/src/client/admin/components/input/style.module.scss

```diff
@@ -0,0 +1,24 @@
+.wrapper {
+	width: 100%;
+	height: calc( var( --spacing-base ) * 5 ); // 40px;
+	padding: var( --spacing-base );
+	border: 1px solid var( --jp-black );
+	border-radius: 4px;
+	background-color: var( --jp-white );
+	display: flex;
+
+	&:focus-within {
+		border-width: 2px;
+		padding: calc( var( --spacing-base ) - 1px ); // 7px;
```

Comment:
We shouldn't do this. Why exactly?

Reply:
To keep the input content static while increasing the border width.
I checked here and `InputControl` uses `box-shadow` instead. I'll change it.

Reply:
Done.